test/models/report_test.rb

```diff
@@ -3,7 +3,19 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test 'user can not edit report' do
+    report = reports(:report_by_alice)
+    assert_not report.editable?(users(:bob))
+  end
+
+  test 'user can edit report' do
+    report = reports(:report_by_alice)
+    assert report.editable?(users(:alice))
+  end
+
+  test 'Get the date when the daily report was created' do
+    report = reports(:report_by_bob)
+    report.created_at = Date.parse('2024-06-11 09:57:39.890045')
```

Comment:
```suggestion
    report.created_at = Date.parse('2024-06-11 09:57')
```

必要以上に細かい値を入れてもあまり意味がない（逆に他の人が何か意味があるのかと勘ぐってしまう）ので、分ぐらいまでで十分だと思います。